system/Cookie/Cookie.php

```diff
@@ -289,9 +286,7 @@ public function getPrefixedName(): string
 		else
 		{
 			$search  = str_split(self::$reservedCharsList);
-			$replace = array_map(static function (string $char): string {
-				return rawurlencode($char);
-			}, $search);
+			$replace = array_map('rawurlencode', $search);
 
 			$name .= str_replace($search, $replace, $this->getName());
 		}
```

Comment:
Short if condition? 
```suggestion
		$name   = $this->getPrefix();
		$search = str_split(self::$reservedCharsList);
		$name  .= $this->isRaw()
			? $this->getName()
			: str_replace($search, array_map('rawurlencode', $search), $this->getName());

		return $name;
```

Reply:
I tried using the short if but it does not look too much understandable.
```php
return $this->getPrefix() . $this->isRaw()
    ? $this->getName()
    : str_replace(
        $search = str_split(self::$reservedCharsList),
        array_map('rawurlencode', $search),
        $this->getName()
    );
```
I would still prefer understandable/readable code rather than concise one.